site/troubleshooting.md

```diff
@@ -290,6 +291,27 @@ Proxy](/site/tasks/weave-docker-api/weave-docker-api.md) will re-attach any appl
 containers that it originally attached to the Weave network when they
 restart.
 
```

Comment:
(Remove wills. also passive voice as a general rule.)

Note that the Weave Net router creates the `weave` network bridge if necessary when it restarts. The [Weave Net Docker API Proxy](/site/tasks/weave-docker-api/weave-docker-api.md) then re-attaches any application
containers that it originally attached to the Weave network when they
restart.